redhat-developer/app-services-cli
# feat(auth): remove double-login for code flow

### Description

Fixes #548 

This PR removes the need to log in manually twice, by performing a login request from the redirect page.

### Verification Steps

1. make sure you are logged out and cannot interact with topics
2. Run `rhoas login` and enter your credentials. You will be redirected twice and will be brought tot he MAS-SSO success page.
3. Run `rhoas kafka topic list` on a Kafka instance that you own. It should work.

### Type of change


### Checklist
